integration-tests/plugins/__tests__/invites/resend-invite.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { IAuthModuleService, IUserModuleService } from "@medusajs/types"
+import { initDb, useDb } from "../../../environment-helpers/use-db"
+
+import { AxiosInstance } from "axios"
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { createAdminUser } from "../../helpers/create-admin-user"
+import { getContainer } from "../../../environment-helpers/use-container"
+import path from "path"
+import { startBootstrapApp } from "../../../environment-helpers/bootstrap-app"
+import { useApi } from "../../../environment-helpers/use-api"
+
+jest.setTimeout(50000)
+
+const env = { MEDUSA_FF_MEDUSA_V2: true }
+const adminHeaders = {
+  headers: { "x-medusa-access-token": "test_token" },
+}
+
+describe("POST /admin/invites/:id/resend", () => {
+  let dbConnection
+  let appContainer
+  let shutdownServer
+  let userModuleService: IUserModuleService
+
+  beforeAll(async () => {
+    const cwd = path.resolve(path.join(__dirname, "..", ".."))
+    dbConnection = await initDb({ cwd, env } as any)
+    shutdownServer = await startBootstrapApp({ cwd, env })
+    appContainer = getContainer()
+    userModuleService = appContainer.resolve(ModuleRegistrationName.USER)
+  })
+
+  beforeEach(async () => {
+    await createAdminUser(dbConnection, adminHeaders)
+  })
+
+  afterAll(async () => {
+    const db = useDb()
+    await db.shutdown()
+    await shutdownServer()
+  })
+
+  afterEach(async () => {
+    const db = useDb()
+    await db.teardown()
+  })
+
+  it("should resend a single invite", async () => {
+    const invite = await userModuleService.createInvites({
+      email: "potential_member@test.com",
+    })
+
+    const api = useApi()! as AxiosInstance
+
+    const response = await api.post(
+      `/admin/invites/${invite.id}/resend`,
+      {},
+      adminHeaders
+    )
+
+    expect(response.status).toEqual(200)
```

Comment:
I would be nice to test that the event was also emitted, as that is probably more important here for now

Reply:
Added a test in the integration test suite that the event is emitted